data/filedefs/filetypes.autoit

```diff
@@ -0,0 +1,72 @@
+# For complete documentation of this file, please see Geany's main documentation
+# AutoIt 3.3.14.2
+[styling]
+# Edit these in the colorscheme .conf file instead
+default=default
+comment=comment
+commentblock=comment
+number=number
+function=function
+keyword=type
+macro=preprocessor
+string=string
+operator=operator
+variable=keyword_2
```

Comment:
```identifier == default``` in all Geany's colorshemes (default and from [here](https://github.com/geany/geany-themes)), but I wanted to use the highlighting.
Some```filetypes*``` has ```variable=type```, but I already use ```type``` and I took ```keyword_2``` because that it no one uses and it looks normal in all colorschemes.

I don't know what to do here.

Reply:
OK… I'm not sure either what to do here, and I guess this is one of the instances where the named styles we provide are not always enough.

The actual problem is possibly more subtle, it `identifier` might be styled as `default` by most colorschemes because doing otherwise results in weird behavior in some filetypes, which might be for a bad reason; or it might be that we don't have enough subtleties in the styles we provide.

@codebrainz do you have an opinion?